### PR TITLE
v1.0.2 release notes

### PR DESCRIPTION
**Pull request by mattcary**
/kind documentation

```release-note
None
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.0.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-# v1.0.1 - Changelog since v1.0.0
+# v1.0.2 - Changelog since v1.0.1
 
-## Changes by Kind
+### Bug or Regression
+
+- Update base image to buster-v.1.9.0 ([#829](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/829), [@mattcary](https://github.com/mattcary))
+4f
```
Comment (saikat-royc) [inline CHANGELOG/CHANGELOG-1.0.md:6]: remove 4f?

**Comment by mattcary** [comment]
oops, thx

**Comment by mattcary** [comment]
Force-pushed as the release-1.0 integration test is broken due to the snapshot (see #829 and #526).

**Pull request merged**